tests/resources/conformance_suites/dba/fr/fr81-invalid.xbrl

```diff
@@ -0,0 +1,557 @@
+    <fsa:ShorttermPayablesToGroupEnterprises contextRef="ID_8" unitRef="DKK" decimals="-3">0000</fsa:ShorttermPayablesToGroupEnterprises>
+    <fsa:ShorttermPayablesToGroupEnterprises contextRef="ID_9" unitRef="DKK" decimals="-3">145000</fsa:ShorttermPayablesToGroupEnterprises>
+    <fsa:ShorttermPrepaymentsReceivedFromCustomers contextRef="ID_8" unitRef="DKK" decimals="-3">35754000</fsa:ShorttermPrepaymentsReceivedFromCustomers>
+    <fsa:ShorttermPrepaymentsReceivedFromCustomers contextRef="ID_9" unitRef="DKK" decimals="-3">36857000</fsa:ShorttermPrepaymentsReceivedFromCustomers>
+    <fsa:ShorttermReceivables contextRef="ID_8" unitRef="DKK" decimals="-3">156590000</fsa:ShorttermReceivables>
+    <fsa:ShorttermReceivables contextRef="ID_9" unitRef="DKK" decimals="-3">72160000</fsa:ShorttermReceivables>
+    <fsa:ShorttermReceivablesFromGroupEnterprises contextRef="ID_8" unitRef="DKK" decimals="-3">101229000</fsa:ShorttermReceivablesFromGroupEnterprises>
+    <fsa:ShorttermReceivablesFromGroupEnterprises contextRef="ID_9" unitRef="DKK" decimals="-3">0000</fsa:ShorttermReceivablesFromGroupEnterprises>
+    <fsa:ShorttermTaxPayables contextRef="ID_8" unitRef="DKK" decimals="-3">9054000</fsa:ShorttermTaxPayables>
+    <fsa:ShorttermTaxPayables contextRef="ID_9" unitRef="DKK" decimals="-3">0000</fsa:ShorttermTaxPayables>
+    <fsa:ShorttermTradePayables contextRef="ID_8" unitRef="DKK" decimals="-3">82585000</fsa:ShorttermTradePayables>
+    <fsa:ShorttermTradePayables contextRef="ID_9" unitRef="DKK" decimals="-3">49416000</fsa:ShorttermTradePayables>
+    <fsa:ShorttermTradeReceivables contextRef="ID_8" unitRef="DKK" decimals="-3">31255000</fsa:ShorttermTradeReceivables>
+    <fsa:ShorttermTradeReceivables contextRef="ID_9" unitRef="DKK" decimals="-3">23751000</fsa:ShorttermTradeReceivables>
+    <fsa:TaxExpense contextRef="ID_0" unitRef="DKK" decimals="-3">57577000</fsa:TaxExpense>
+    <fsa:TaxExpense contextRef="ID_10" unitRef="DKK" decimals="-3">1708000</fsa:TaxExpense>
+    <fsa:TransferredToFromRetainedEarnings contextRef="ID_0" unitRef="DKK" decimals="-3">-23795000</fsa:TransferredToFromRetainedEarnings>
+    <fsa:TransferredToFromRetainedEarnings contextRef="ID_10" unitRef="DKK" decimals="-3">-12946000</fsa:TransferredToFromRetainedEarnings>
+    <gsd:AddressOfSubmittingEnterprisePostcodeAndTown contextRef="ID_0">København K, 1304</gsd:AddressOfSubmittingEnterprisePostcodeAndTown>
+    <gsd:AddressOfSubmittingEnterpriseStreetAndNumber contextRef="ID_0">Adelgade 15</gsd:AddressOfSubmittingEnterpriseStreetAndNumber>
+    <gsd:DateOfFoundationOfReportingEntity contextRef="ID_0">1983-12-23</gsd:DateOfFoundationOfReportingEntity>
+    <gsd:DateOfGeneralMeeting contextRef="ID_0">2023-03-23</gsd:DateOfGeneralMeeting>
+    <gsd:EmailOfReportingEntity contextRef="ID_0">arp-hansen@arp-hansen.dk</gsd:EmailOfReportingEntity>
```

Comment:
Arbitrary line: Would be good to have one fact with a lang attribute with a value other than `en` or `da` so the testcase confirms the validation is checking the language value correctly.